### PR TITLE
Put functions for binary and decimal

### LINKED ISSUE #4: Basic screen drawing
- Drawing pixels to framebuffer
- Printing characters (put_char, put_string, put_decimal, put_hex, put_binary)

### PR DESCRIPTION
**Pull request by antlilja**
Closes #4

### CODE REVIEW

File: kernel/include/rendering.h [hunk 1]
```diff
@@ -37,6 +37,14 @@ void put_binary_len(uint64_t value, uint64_t x, uint64_t y, uint8_t bits);
 void put_binary_32(uint64_t value, uint64_t x, uint64_t y);
 void put_binary_64(uint64_t value, uint64_t x, uint64_t y);
 
+// Put value in decimal at character coordinates
+// returns the number of characters written (including '-')
+uint8_t put_int(int64_t value, uint64_t x, uint64_t y);
```
Comment (morri2) [inline kernel/include/rendering.h]: why is return of put_int a u8 when put_hex and put_binary return u64?

Comment (antlilja) [reply kernel/include/rendering.h]: put_int will never print more than about 20 characters, the capacity of a u64 is therefore unnecessary (the same is true for put_binary and put_hex, but I must've missed put_binary). In reality it doesn't really matter and we could use u64 instead, although it should probably be consistent.

Comment (antlilja) [reply kernel/include/rendering.h:42]: I'll change it to use size_t instead, for functions added in this PR and then make a separate commit to main where I fix it for the other put_ functions

Comment (antlilja) [reply kernel/include/rendering.h]: I'll just use uint64_t for consistency.

**Pull request merged**